src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp

```diff
@@ -1307,5 +1313,11 @@ const MCHAR* AppleseedRendererClassDesc::InternalName()
 
 const MCHAR* AppleseedRendererClassDesc::GetRsrcString(INT_PTR id)
 {
-    return GetString(static_cast<int>(id));
+    const asf::KeyValuePair<int, const wchar_t*>* dialog_string_pair =
+        lookup_kvpair_array(
+            g_dialog_strings,
+            11,
```

Comment:
This number should probably be replaced with constant. I'll fix it if everything else is fine.

Reply:
You can use `LOOKUP_KVPAIR_ARRAY()` which will infer the number of values in `g_dialog_strings` automatically.